mm2src/coins/eth.rs

```diff
@@ -59,6 +59,10 @@ use web3::types::{Action as TraceAction, BlockId, BlockNumber, Bytes, CallReques
 use web3::{self, Web3};
 use web3_transport::{EthFeeHistoryNamespace, Web3Transport, Web3TransportNode};
 
+use crate::utxo::utxo_builder::UtxoConfError;
+use crate::utxo::utxo_common::{CheckPaymentSentError, ExtractSecretError, SendRawTxError, ValidatePaymentError};
+use crate::{FoundSwapTxSpendErr, MmAddressError, ValidateSwapTxError};
```

Comment:
`use crate::` is the same as `use super::`. Please move `FoundSwapTxSpendErr` and `MyAddressError` below.